src/components/views/elements/Slider.tsx

```diff
@@ -83,18 +89,73 @@ export default class Slider extends React.Component<IProps> {
 
         let selection = null;
 
+        const onDrag = (e: React.DragEvent) => {
+            //  console.log(e.target.parentNode.parentNode)
+
+            const target = e.target as HTMLElement;
+            if (!target.parentNode) return null;
+            const parent = target.parentNode as HTMLElement;
+            if (!parent.parentNode) return null;
+            const slider = parent.parentNode as HTMLElement;
```

Comment:
We should use the [`non-null` operand](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator) here

```suggestion
            const slider = parent.parentNode!;
```